### PR TITLE
added custom mobs

### LINKED ISSUE #1: [FEATURE]: Custom mob behavior
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I would like custom mob behavior for the following mobs:
- Cow
- Iron Golem
- Pig
- Pig Zombie
- Sheep
- Wolf
The hostile mobs should be made stronger, the neutral mobs wolf, iron golem and pig zombie should become hostile,
the passive mobs should run away from the player at all times with enhanced speed.

**Describe alternatives you've considered**
X

**Additional context**
Pig Zombies should be given a new gold sword.

### PR DESCRIPTION
**Pull request by Pjiwm**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

*Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.*

**Please provide enough information so that others can review your pull request:**
Added custom mob behavior for the following mobs:

- [x] sheep  
- [x] cow
- [x] iron golem
- [x] pig
- [x] pig zombie
- [x] wolf

**Explain the **details** for making this change. What existing problem does the pull request solve?**
Changed behavior for mobs to increase game difficulty.
passive mobs now run faster and are always on the run for players.
neutral mobs are angry towards the player at all times.

**Test plan (required):**
Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request has visual changes.
Summon custom mob.

**Closing issues**
#1 

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

**Comment by Pjiwm** [comment]
Implemented custom mobs:
- sheep
-  cow
-  iron golem
-  pig
-  pig zombie
-  wolf
All mobs are implemented correctly and have the appropriate behavior
closes #1

File: src/main/java/nl/pjiwm/kaizominecraft/mobs/CustomWolf.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import net.minecraft.server.v1_16_R3.PathfinderGoalNearestAttackableTarget;
+import org.bukkit.Location;
+import org.bukkit.craftbukkit.v1_16_R3.CraftWorld;
+
+public class CustomWolf extends EntityWolf {
+    public CustomWolf(Location location) {
+        super(EntityTypes.WOLF, ((CraftWorld)location.getWorld()).getHandle());
+        this.setPosition(location.getX(), location.getY(), location.getZ());
+        this.targetSelector.a(0, new PathfinderGoalNearestAttackableTarget<>(this, EntityPlayer.class, true));
+
```
Comment (KaanMol) [inline src/main/java/nl/pjiwm/kaizominecraft/mobs/CustomWolf.java]: Maybe remove the extra spaces?

File: src/main/java/nl/pjiwm/kaizominecraft/mobs/CustomPigZombie.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import org.bukkit.Location;
+import org.bukkit.craftbukkit.v1_16_R3.CraftWorld;
+
+public class CustomPigZombie extends EntityPigZombie {
+    public CustomPigZombie(Location location) {
+        super(EntityTypes.ZOMBIFIED_PIGLIN, ((CraftWorld)location.getWorld()).getHandle());
+        this.targetSelector.a(0, new PathfinderGoalNearestAttackableTarget<>(this, EntityPlayer.class, true));
+        this.setPosition(location.getX(), location.getY(), location.getZ());
+    }
+
```
Comment (KaanMol) [inline src/main/java/nl/pjiwm/kaizominecraft/mobs/CustomPigZombie.java:16]: Also spaces here

File: src/main/java/nl/pjiwm/kaizominecraft/mobs/CustomIronGolem.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+import net.minecraft.server.v1_16_R3.EntityIronGolem;
+import net.minecraft.server.v1_16_R3.EntityPlayer;
+import net.minecraft.server.v1_16_R3.EntityTypes;
+import net.minecraft.server.v1_16_R3.PathfinderGoalNearestAttackableTarget;
+import org.bukkit.Location;
+import org.bukkit.craftbukkit.v1_16_R3.CraftWorld;
+
+public class CustomIronGolem extends EntityIronGolem {
+
```
Comment (KaanMol) [inline src/main/java/nl/pjiwm/kaizominecraft/mobs/CustomIronGolem.java]: More empty spaces

**Review by KaanMol — CHANGES_REQUESTED** [review]
A lot of empty spaces, can you remove them?

**Review by KaanMol — APPROVED** [review]

**Pull request merged**